modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Default.js

```diff
@@ -0,0 +1,64 @@
+// If a style has been marked `def`, we need to make sure it
+// and it's attributes are prioritized in our editor.
+// It's easier to create a new Node with our specifications and put it
+// at the front of the line than try to infer between editor instantiation
+// and the editor-user setting styles via the toolbar to know when its needed.
+
+import { Node } from '@tiptap/core';
+import Heading from '@tiptap/extension-heading';
+import Paragraph from '@tiptap/extension-paragraph';
+
+const nodeMap = {
+  heading: Heading,
+  paragraph: Paragraph
+};
+
+export default (options) => {
+  const def = options.styles.filter(style => style.def)[0];
+
+  // Configuration has a default style
+  if (def) {
```

Comment:
It's OK to return nothing here when there's no default?

Reply:
We do ensure we have a `def` style even if we set it, but should the checks for a `def` style be removed to make that clearer?

where we make sure have a default 👉  https://github.com/apostrophecms/apostrophe/pull/3423/files#diff-19bda1b8bcbb44e43a07e5665b4030fc2604c5fe3fc632974c3b25289d9a00c1R256-R266